client/lib/purchases/index.js

```diff
@@ -147,19 +147,19 @@ function isPaidWithPaypal( purchase ) {
 }
 
 function isPaidWithIdeal( purchase ) {
-	return 'ideal' === purchase.payment.type.toLowerCase();
+	return 'ideal' === purchase.payment.type;
```

Comment:
Are we sure that `type` will always be lower case?

If not, should we be doing something like this:

```
return purchase.payment.type !== undefined && 'ideal' == purchase.payment.type.toLowerCase();
```

Reply:
It is currently always lower case, but I'll make another iteration using your suggestion which is more robust and future-proof. Thanks!